src/LiquidationPairFactory.sol

```diff
@@ -21,11 +21,6 @@ contract LiquidationPairFactory {
     /* ============ Variables ============ */
     LiquidationPair[] public allPairs;
```

Comment:
I'm thinking this should be a mapping so that we can verify if a pair is made by this factory, yeah?

Reply:
Done: https://github.com/pooltogether/v5-liquidator/pull/2/commits/1e2b0ea029c0bf62b93ba074887e6e3fa7a61b6e